Lib/test/test_sys.py

```diff
@@ -1032,6 +1032,10 @@ def test_implementation(self):
         self.assertEqual(sys.implementation.name,
                          sys.implementation.name.lower())
 
+        if hasattr(sys.implementation, '_architecture'):
+            self.assertIn(sys.implementation._architecture,
+                          ['win32', 'amd64', 'arm32', 'arm64', ''])
```

Comment:
Hmm, in what case can this be `''`?

Reply:
When the system is unknown, he will be an empty string.

Reply:
Is that even possible? Windows only supports these four architectures AFAIK, it should be available information.

Reply:
I have answered victor in the above discussion. You can refer to the above answer.

In fact, this is related to `python.prop`.

Reply:
I don't think that this case can happen and I suggest to remove it.